Upstream-Status:Inappropriate [embedded specific]
Signed-off-by: Nitin A Kamble <nitin.a.kamble@intel.com>
2011/07/01

Fix these Package QA warnings before they are converted into fetal errors:

WARNING: QA Issue: package perl-module-compress contains bad RPATH /build_disk/poky_build/build0/tmp/sysroots/qemux86/usr/lib in file /build_disk/poky_build/build0/tmp/work/i586-poky-linux/perl-5.12.3-r1/packages-split/perl-module-compress/usr/lib/perl/5.12.3/auto/Compress/Raw/Zlib/Zlib.so

This fixes this warning for perl recipe as well as libxml-parser-perl recipe. 
It is a fix to MakeMaker within perl, so all such perl recipes will get 
fixed with this perl fix.

Index: perl-5.14.2/cpan/ExtUtils-MakeMaker/lib/ExtUtils/Liblist/Kid.pm
===================================================================
--- perl-5.14.2.orig/cpan/ExtUtils-MakeMaker/lib/ExtUtils/Liblist/Kid.pm
+++ perl-5.14.2/cpan/ExtUtils-MakeMaker/lib/ExtUtils/Liblist/Kid.pm
@@ -55,6 +55,7 @@ sub _unix_os2_ext {
     my($found) = 0;
 
     # Debian-specific: don't use LD_RUN_PATH for standard dirs
+    push(@libpath, "SYSROOTLIB");
     $ld_run_path_seen{$_}++ for @libpath;
 
     foreach my $thislib (split ' ', $potential_libs) {
